it/utils/TaxYearUtils.scala

```diff
@@ -20,6 +20,8 @@ import play.api.Logging
 
 object TaxYearUtils extends Logging {
```

Comment:
why do we have this class in test? and it is repeated twice

Reply:
Resolved